File: app/src/main/java/com/ps/simplepapersoccer/Activities/TutorialPagerAdapter.java
package com.ps.simplepapersoccer.Activities;

import android.app.Activity;
import android.content.Context;
import android.media.MediaPlayer;
import android.net.Uri;
import android.support.v4.content.ContextCompat;
import android.support.v4.view.PagerAdapter;
import android.support.v4.view.ViewPager;
import android.view.Gravity;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ImageView;
import android.widget.TextView;
import android.widget.VideoView;

import com.ps.simplepapersoccer.R;

public class TutorialPagerAdapter extends PagerAdapter {
    Context mContext;
    Activity activity;

    TutorialPagerAdapter(Context context, Activity activity) {
        this.mContext = context;
        this.activity = activity;
    }

    @Override
    public int getCount() {
        return sliderImagesId.length;
    }

    private int[] sliderImagesId = new int[]{ R.drawable.tutorial1, R.drawable.tutorial2, R.drawable.tutorial3, R.drawable.tutorial4 };

    @Override
    public boolean isViewFromObject(View v, Object obj) {
        return v == obj;
    }

    @Override
    public Object instantiateItem(ViewGroup container, int i) {
        ImageView imageView = new ImageView(mContext);

        imageView.setImageDrawable(ContextCompat.getDrawable(mContext, sliderImagesId[i]));

        ((ViewPager) container).addView(imageView);
        return imageView;
    }

    @Override
    public void destroyItem(ViewGroup container, int i, Object obj) {
        ((ViewPager) container).removeView((ImageView) obj);
    }
}

File: app/src/main/java/com/ps/simplepapersoccer/Activities/TutorialPagerAdapter.kt
package com.ps.simplepapersoccer.activities

import android.app.Activity
import android.content.Context
import androidx.core.content.ContextCompat
import androidx.viewpager.widget.PagerAdapter
import androidx.viewpager.widget.ViewPager
import android.view.View
import android.view.ViewGroup
import android.widget.ImageView

import com.ps.simplepapersoccer.R

class TutorialPagerAdapter internal constructor(private var context: Context) : PagerAdapter() {

    override fun getCount(): Int {
        return sliderImagesId.size
    }

    private val sliderImagesId = intArrayOf(R.drawable.tutorial1, R.drawable.tutorial2, R.drawable.tutorial3, R.drawable.tutorial4)

    override fun isViewFromObject(v: View, obj: Any): Boolean {
        return v == obj
    }

    override fun instantiateItem(container: ViewGroup, i: Int): Any {
        val imageView = ImageView(context)

        imageView.setImageDrawable(ContextCompat.getDrawable(context, sliderImagesId[i]))

        (container as ViewPager).addView(imageView)
        return imageView
    }

    override fun destroyItem(container: ViewGroup, i: Int, obj: Any) {
        (container as ViewPager).removeView(obj as ImageView)
    }
}
